### ISSUE TITLE
Screenshot not working properly, images with blue masks are saved

### ISSUE THREAD

**Issue by phoe** [linuxmint/Cinnamon#8089]
* Cinnamon 3.8.8 64-bit on debian sid
 * NVidia blob driver 390.87
 * [~/.xsession-errors](https://github.com/linuxmint/Cinnamon/files/2635631/default.xsession-errors.txt)

**Issue**

When taking screenshots of a certain area, the area will be covered by a blue mask as a visual effect, and the mask is often captured together with the screenshot.

Similar to https://github.com/linuxmint/Cinnamon/issues/4605

**Steps to reproduce**

1. Press Shift + PrntScr button 
2. Using the mouse, make some selection, 
3. Release the left click button.

**Expected behaviour**

Screenshot's colors are not modified.

**Other information**

Example of photo I want to take a screenshot of: 
![84lcctm2lxv11](https://user-images.githubusercontent.com/15045546/49324655-80a0ec80-f532-11e8-9675-b68a7679b3ae.jpg)

Example of affected screenshot: 
![zrzut ekranu z 2018-12-01 06-23-46](https://user-images.githubusercontent.com/15045546/49324632-d9bc5080-f531-11e8-9459-c36c8adecc66.png)

**Comment by ManuelSchneid3r** [comment]
This looks like the usual selection rectangle. I dont have this problem on arch, Cinnamon 4.0.6, gnome-screenshot 3.30.0

**Comment by phoe** [comment]
Yes, this *is* the selection rectangle. The issue is that it is captured on top of the screenshot.

This issue is intermittent which makes me think it is a race condition of some sort.

**Comment by Strum355** [comment]
Still have this in cinnamon 4.0.9 and gnome-screenshot 3.30.0, Antergos

**Comment by phoe** [comment]
Still affecting me with the current debian sid version.

**Comment by Vahan86** [comment]
I can't reproduce the issue in Cinnamon 4.4.8, LM 19.3

**Comment by Strum355** [comment]
Still happening on Cinnamon 4.4.9 gnome-screenshot 3.34.0

**Comment by Vahan86** [comment]
@icarter09, can we mark this issue as a bug?

**Comment by leigh123linux** [comment]
This is a known gnome-screenshot issue

https://gitlab.gnome.org/GNOME/gnome-screenshot/-/issues/56
